vipps-login-api-faq.md

```diff
@@ -404,6 +405,14 @@ You can do it yourself on
 [portal.vipps.no](https://portal.vipps.no/).
 Press `rediger`/`edit` under `salgsstedsinfo`/`?` and change to the desired name.
 
+## How can I logout a user?
+
+The answer here is twofold; _Vipps Login_ doesn't support logout from the browser, but you are free to log the user out of your service.  
+You can't log the user out of the current browser because it would lead to a poor end user experience.  
```

Comment:
"[...] because it would lead to a poor end user experience because the user would then also be logged out everywhere else"?

Reply:
Skip "current".

Reply:
"The reason for not allowing logout in the browser is because this would lead to a poor user experience due to the user also being logged out everywhere else." ?
Don't really like using because twice in the same sentence.
Kinda attached to "current", because it's actually quite important technically. You wouuld log the person out in one browser, not all the browsers.